addons/mail/static/src/models/rtc_call_participant_card.js

```diff
@@ -104,6 +121,28 @@ registerModel({
                 return this.invitedGuest.name;
             }
         },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeOutboundConnectionTypeText() {
+            if (!this.rtcSession || !this.rtcSession.localCandidateType) {
+                return this.env._t("no connection");
+            }
+            return sprintf(
+                this.env._t('Outbound: %(candidateType)s (%(protocol)s)'), {
```

Comment:
Note that I still don't understand and find confusing whether inbound and outbound are related to the the current partner or the participant of the card.